styles.css

```diff
@@ -562,3 +578,250 @@ hr {
+    align-items: center;
+    margin-right: 143px;
+    margin-left: 141px;
+    gap: 5px;
+  }
+
+  .recent-work {
+    grid-column: 1/4;
+    grid-template-columns: 0.25fr 0.75fr;
+    width: 100%;
+  }
+
+  .recent-work h2 {
+    width: 327px;
+    margin-left: 0%;
+  }
+
+  .recent-work hr {
+    height: 1.5px;
+    margin: 28px 10% 0% 2%;
+    width: 90%;
+  }
+
+  .post-stories {
+    grid-column: 1/4;
+    flex-direction: row;
+    gap: 20px;
+    margin-bottom: 24px;
+  }
+
+  .post-stories img {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    gap: 10px;
+  }
+
+  .multi-post {
+    display: flex;
+    flex: 1;
+    flex-direction: column;
+    justify-content: flex-start;
+    width: 30%;
+    gap: 10px;
+    row-gap: 24px;
+  }
+
+  .multi-post h2 {
+    font-family: "Crete Round", sans-serif;
+    font-size: 40px;
+    line-height: 52px;
+    margin: -10px;
+    margin-bottom: 10px;
+  }
+
+  .multi-post p {
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    padding-right: 15%;
+  }
+
+  .post-stories ul {
+    margin-top: -10px;
+  }
+
+  .post-stories button {
+    padding: 12px 14px;
+  }
+
+  .card {
+    height: 435px;
+    display: grid;
+  }
+
+  .card-hover:hover {
+    border: 3px solid #0d151e;
+    cursor: pointer;
+    width: 328px;
+    height: 435px;
+  }
+
+  .btn2 {
+    display: none;
+  }
+
+  .about-me {
+    background: #000;
+    width: 100%;
+    background-image: url(img/Frame-1.png);
+    background-repeat: no-repeat;
+    background-position: top 15% right 0%;
+    background-size: 35%;
+  }
+
+  .about h2,
+  .about p,
+  .btn3 {
+    margin-left: 10%;
+  }
+
+  .about p {
+    width: 36%;
+  }
+
+  .hr2 {
+    width: 80%;
+    margin-left: 10%;
+    margin-top: 10%;
+  }
+
+  .exp {
+    display: grid;
+    width: 100%;
+    grid-template-columns: 25% 25% 25%;
+    margin-left: 0%;
+  }
+
+  .contact {
+    display: flex;
+    width: 100%;
+    margin-left: 0%;
+  }
+
+  .form-title {
+    width: 455px;
+    margin: 10% 10% 10% 0%;
+    text-align: start;
+    margin-left: 135px;
+  }
+
+  .contact-form {
+    background-color: #ffff;
+  }
+
+  .form {
+    width: 40%;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    margin: 8% 15% 0% 0%;
+    padding: 0%;
+  }
+
+  .form-input {
+    margin: 0 24px;
+    padding: 16px;
+    display: flex;
+    align-items: flex-start;
+  }
+
+  .textmsg {
+    width: 600px;
+    height: 114px;
+    margin-top: 24px;
+    margin-left: 24px;
+    padding: 12px;
+  }
+
+  .form input:nth-child(1) {
+    width: 600px;
+    margin-left: 20px;
+  }
+
+  .submit-btn {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    align-items: flex-start;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #fff;
+    padding: 12px;
+    margin-left: -60%;
+  }
```

Comment:
- [ Required ] Please consider aligning the `submit` button with the other form inputs as illustrated in the Figma design. Please see the screenshot below for more details on this.

## Figma template
![Screenshot from 2023-01-20 11-11-41](https://user-images.githubusercontent.com/16392046/213647549-bddc938d-462c-4245-a088-f92444e14aab.png)

## Your design 
![Screenshot from 2023-01-20 11-03-48](https://user-images.githubusercontent.com/16392046/213647643-af6ae901-d018-4342-9662-dae2a88500af.png)